docs/howto/openstack/nova/resize-server.md

```diff
@@ -0,0 +1,125 @@
+
+    ![server-edit](assets/resize-server/04-modify-server.png)
+=== "OpenStack CLI"
+    To list all available flavors you can simply run `openstack flavor list`, but that will return a very long unsorted list, instead we recommend the following command:
+
+    ```bash
+    openstack flavor list -c Name -f value | grep '[0-9]c[0-9]' | sort -V
+    ```
+
+    The printout is a simple and clean list sorted by the compute type, the number of cores and then by the amount of memory.
+
+    ```plain
+    X.1c1gb
+    X.1c2gb
+    X.1c4gb
+    X.2c2gb
+    X.2c4gb
+    X.2c8gb
+    X.2c16gb
+    X.4c4gb
+    ...
+    ```
+
+## Initiating the resize
+
+Choose a new flavor that you want your server to use instead.
+
+=== "{{gui}}"
+    Once selected, the _Resize_ button will appear.
+
+    ![resize-button](assets/resize-server/05-resize-button.png)
+
+    Click on it to start the resize.
+
+    While the resize is ongoing you will see a spinning circle saying _Resize is in progress_.
+
+    ![resize-in-progress](assets/resize-server/06-resize-in-progress.png)
+=== "OpenStack CLI"
+    To start the resize use the following command:
+
+    ```bash
+    openstack server resize --flavor <new_flavor> <server_id>
+    ```
+
+    While the resize is ongoing the server should have the OS-EXT-STS:task_state of `resize_migrating` and the status of `RESIZE`.
+
+    ```bash
+    openstack server show -c OS-EXT-STS:task_state -c OS-EXT-STS:vm_state -c status <server_id>
+    ```
+
+    ```plain
+    +-----------------------+------------------+
+    | Field                 | Value            |
+    +-----------------------+------------------+
+    | OS-EXT-STS:task_state | resize_migrating |
+    | OS-EXT-STS:vm_state   | active           |
+    | status                | RESIZE           |
+    +-----------------------+------------------+
+    ```
+
+    You may proceed with the next step once your server status is `VERIFY_RESIZE`
+
+    ```plain
+    +-----------------------+---------------+
+    | Field                 | Value         |
+    +-----------------------+---------------+
+    | OS-EXT-STS:task_state | None          |
+    | OS-EXT-STS:vm_state   | resized       |
+    | status                | VERIFY_RESIZE |
+    +-----------------------+---------------+
+    ```
+
+The resize process might take a minute or more. {{brand}} Cloud will now make a restore point incase the resize process fails, it would then restore your server to the state it was before the resize.
+
+## Confirming the resize
+
+Your server is now using the new flavor you selected earlier and you need to make sure the server is working as intended after the resize.
+
+Once you are certain you server is working as intended, you **must** confirm the resize. If you fail to confirm the resize, your server will have the restore point created by the resize process left behind and this can cause your server to be reverted to this point at a later date, resulting in **data loss**.
```

Comment:
Well, you actually should - not must. As resize will be auto-confirmed in case it has not be reverted in 24hours.
Also this side effect for reverting is valid only for non-bfv which is not the way we expect new servers to be created and usage of non-bfv should reduced as much as possible anyway. Moreover, we provide instruction that explicitly fetches only bfv flavors.

Reply:
oh really, there's no roll back when reverting while using BFV. I did not know that, that's great. Will rewrite this whole section.